src/components/organisms/AdminVenueView/components/RunTabUserInfo/RunTabUserInfo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+
+import { useWorldUserLocation } from "hooks/users";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+export interface RunTabUserInfoProps {
+  user: WithId<User>;
+}
+
+export const RunTabUserInfo: React.FC<RunTabUserInfoProps> = ({ user }) => {
+  const { userLocation } = useWorldUserLocation(user?.id);
+  const userLastSeenIn = userLocation?.lastSeenIn;
+
+  const location = userLastSeenIn ? Object.keys(userLastSeenIn)?.[0] : "";
```

Comment:
are we sure that the result is always in the 1st key?

Reply:
Yes, you can be only in one location and it's always the first key.